modules/media/php/mediafileprovisioner.class.inc

```diff
@@ -38,22 +38,23 @@ class MediaFileProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
     {
         parent::__construct(
             "SELECT m.file_name as fileName,
-                    c.PSCID as pscid,
-                    s.Visit_label as visitLabel,
-                    l.language_label as languageLabel,
-                    (SELECT Full_name FROM test_names
-                      WHERE Test_name=m.instrument) as fullName,
-               		s.CenterID as centerId,
-               		s.ProjectID as projectId,
-               		m.uploaded_by as uploadedBy,
-               		m.date_taken as dateTaken,
-               		substring(m.comments, 1, 50) as comments,
-                    m.last_modified as lastModified,
-               		m.file_type as fileType,
-               		s.CandID as candidateId,
-               		s.ID as sessionId,
-               		m.hide_file as hideFile,
-               		m.id
+                c.PSCID as pscid,
+                s.Visit_label as visitLabel,
+                l.language_label as languageLabel,
+                (SELECT Full_name FROM test_names
+                  WHERE Test_name=m.instrument) as fullName,
+                s.CenterID as centerId,
+                s.ProjectID as projectId,
+                m.uploaded_by as uploadedBy,
+                m.date_taken as dateTaken,
+                substring(m.comments, 1, 50) as comments,
+                m.last_modified as lastModified,
+                m.file_type as fileType,
+                s.CandID as candidateId,
+                s.ID as sessionId,
+                m.hide_file as hideFile,
+                m.id,
+                m.language_id as language
```

Comment:
language label is already being queried on line 43.

why is this queried here ? and why use `language` when everywhere else changed to either `language_id` or `languageLabel` ?

Reply:
That line might be extra. I had trouble finding the exact place to correct for the edit window.